vcfconcat.c

```diff
@@ -549,6 +553,114 @@ static void concat(args_t *args)
     }
 }
 
+BGZF *hts_get_bgzfp(htsFile *fp);
```

Comment:
No.  Use the declaration in tbx.h, with a note as `#include <htslib/tbx.h> // for hts_get_bgzfp()`